product/urls.py

```diff
@@ -1,6 +1,14 @@
 from django.urls    import path
-from product.views  import ProductDetailView
+from product.views  import (
+                            ProductDetailView,
+                            ProductListView,
+                            MdChoiceView,
+                            CategoryView,
+                            )
```

Comment:
```suggestion
from product.views  import (
    ProductDetailView,
    ProductListView,
    MdChoiceView,
    CategoryView,
)
```
한 탭 정도 띄우시면 가장 적절할 것 같습니다

Reply:
align에 너무 목숨걸었네요... 수정했습니다! 👍